aiohttp/client_reqrep.py

```diff
@@ -547,8 +547,8 @@ def update_proxy(
         proxy_auth: Optional[BasicAuth],
         proxy_headers: Optional[LooseHeaders],
     ) -> None:
-        if proxy and not proxy.scheme == "http":
-            raise ValueError("Only http proxies are supported")
+        # if proxy and not proxy.scheme == "http":
+        #     raise ValueError("Only http proxies are supported")
```

Comment:
This will also need to be reverted before merge.

Reply:
Agreed. I think my next push will remove it after I can see that even the http endpoints through https work. That way the xfail can be confident it's due to actual breakage and not this check failing the test.